BootloaderCorePkg/Tools/GenCapsuleFirmware.py

```diff
@@ -333,6 +334,8 @@ def DumpInfo(self):
             FmpCapsuleImageHeader.DumpInfo('  ')
             print('')
 
+def UnicodeStrToHexString(Str):
+    return (binascii.hexlify(Str.encode('utf-8'))).decode()
 

```

Comment:
Is the str really unicode string? or just utf-8 string.

Reply:
in Python 3 all strings are unicode strings.